db/migrate/20210910170107_add_response_sets_loans_foreign_key.rb

```diff
@@ -0,0 +1,11 @@
+class AddResponseSetsLoansForeignKey < ActiveRecord::Migration[6.1]
+  def change
+    reversible do |dir|
```

Comment:
same here, if the up/down code is different, it's easier to read if they are separate methods.